src/app/components/card/CardButton.vue

```diff
@@ -8,22 +8,35 @@
     is-external-icon-disabled
     :to="props.to"
   >
-    <Card>
+    <Card :background-color="backgroundColor">
       <div class="flex items-center gap-4">
-        <div class="px-2">
+        <div class="flex items-center px-2">
           <slot />
         </div>
         <div class="flex flex-1 flex-col">
-          <span class="font-bold">{{ title }}</span>
+          <span :class="['font-bold', textWhite ? 'text-white' : 'text-black']">
+            {{ title }}
+          </span>
           <span v-if="description" class="opacity-60">
             {{ description }}
           </span>
         </div>
         <div class="opacity-60">
           <slot v-if="$slots.iconSecondary" name="iconSecondary" />
-          <div v-else-if="props.to">
-            <IHeroiconsArrowTopRightOnSquare v-if="isExternal" />
-            <IHeroiconsChevronRight v-else />
+          <div v-else-if="!$slots.iconSecondary">
```

Comment:
You should be able to pass in the secondary icon in the [slot](https://vuejs.org/guide/components/slots.html) here instead of a prop. (Also

```suggestion
          <div v-else>
```
should be enough.)

Reply:
I've pushed the change. Please review, you can resolve this comment when you understand the change I pushed. If not, feel free to ask questions.